src/lib/auth.ts

```diff
@@ -28,24 +48,36 @@ const createAuth = () => {
     set({ user: null, known: true })
   }
 
+  // This is a helper function to get the auth provider for login.
+  // It will not be avialble in the export.
   function providerFor(name: string) {
     switch (name) {
       case 'google':   return new GoogleAuthProvider()
       default:         throw 'unknown provider ' + name
     }
   }
 
+  // This function will be available as `auth.signInWith(providerName)`
+  // Calling it will trigger an authStateChange event, causing
```

Comment:
This is describing how firebase works, I think all that should be documented is how to call what this module provides